arch/arm64/boot/dts/rockchip/Makefile

```diff
@@ -343,7 +343,7 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588-armsom-w3.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588-firefly-itx-3588j.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588-fxblox-rk1.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588-hinlink-h88k.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588-nanopc-cm3588-nas.dtb
```

Comment:
I don't think this is a backward-compatibility change. What about adding fallback devicetree for the old naming?

Reply:
> @efectn Backwards compatible in which way do you mean?

Systems using this devicetree name will not boot anymore after updating kernel